### PR TITLE
nojira | adjust summer course card and page; fix carousel arrow focus state

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- add course syllabus link
- add margin bottom to apply now link
- adjust summer course teaser card
- update carousel arrow focus

# Review By (Date)
- When possible

# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to preview
2. Verify that the course syllabus link conditionally appears and that there is spacing at the bottom of the course page
3. Verify fix for the carousel arrows focus state bug
4. Verify that the course teaser card appears as expected in carousel
5. Review code

### CODE REVIEW

File: src/components/nodes/cards/summer-course/summer-course-card.tsx [hunk 1]
```diff
@@ -19,9 +20,16 @@ const SummerCourseCard = ({node, headingLevel, ...props}: Props) => {
...
-        <Link href={node.path}>{node.title}</Link>
-      </Heading>
+      <div className="flex flex-col">
+        <Heading className="type-3 order-2 mb-0" id={node.id}>
+          <ActionLink className="font-roboto font-normal" href={node.path}>
+            {node.title}
+          </ActionLink>
+        </Heading>
+        {node.sumCourseCatalogNumber && (
+          <div className="rs-mb-0 order-1 font-normal">{node.sumCourseCatalogNumber}</div>
```
Comment (rebeccahongsf) [inline src/components/nodes/cards/summer-course/summer-course-card.tsx:30]: Hmm. I don't see the `node.sumCourseCatalogNumber` being passed in. Do we have a different node data config for teasers? 🤔

File: src/components/patterns/hero-banner.tsx [hunk 2]
```diff
@@ -67,7 +67,7 @@ const HeroBanner = ({
       {children && (
         <div
           className={twMerge(
-            "rs-px-5 rs-py-8 relative order-1 flex w-full flex-col bg-white lg:z-10 lg:w-[calc(50%_-_5rem)]",
```
Comment (rebeccahongsf) [inline src/components/patterns/hero-banner.tsx:70]: This is to fix this:
image

With the min height:
image

File: src/components/nodes/cards/summer-course/summer-course-card.tsx (hunk 1 above)
Comment (pookmish) [reply src/components/nodes/cards/summer-course/summer-course-card.tsx:30]: You have to add the desired fields to the [Graphql query](https://github.com/SU-SWS/summer-nextjs/blob/1.x/src/lib/gql/fragments-node.drupal.gql#L328) and then run `yarn graphql`

File: src/components/nodes/pages/summer-course/summer-course-page.tsx [hunk 3]
```diff
@@ -123,6 +124,14 @@ const SummerCoursePage = async ({node, ...props}: Props) => {
                     node.sumCourseCrossListing.join(", ")}
                 </div>
               )}
+
+              {node.sumCourseSyllabusFile && (
+                <div>
+                  <Link href={node.sumCourseSyllabusFile.mediaFile.url}>
```
Comment (pookmish) [inline src/components/nodes/pages/summer-course/summer-course-page.tsx:130]: let's set `prefetch={false}`

File: src/components/patterns/hero-banner.tsx (hunk 2 above)
Comment (pookmish) [reply src/components/patterns/hero-banner.tsx:70]: I thought that was the desired display.

**Review by pookmish — CHANGES_REQUESTED** [review]
add the GQL fields and any additional fields for the cards.

**Review by pookmish — APPROVED** [review]

**Pull request merged**